polygon/bor/span_id.go

```diff
@@ -33,3 +33,10 @@ func IsBlockInLastSprintOfSpan(blockNum uint64, config *borcfg.BorConfig) bool {
 	startBlockNum := endBlockNum - sprintLen + 1
 	return startBlockNum <= blockNum && blockNum <= endBlockNum
 }
+
+// IsFirstSprintStart returns true if the block num is first block of first sprint
+// E.g. for sprint length 16, it will return true only for block 16. This is
+// to handle a special case for fetching 1st span.
+func IsFirstSprintStart(blockNum uint64, config *borcfg.BorConfig) bool {
```

Comment:
btw the name does not match the logic - the logic is representing `IsSprintStart`
you may need to add some more logic for it to be `IsFirstSprintStart`